src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/AutomaticOrdering/AutomaticOrdering.jsx

```diff
@@ -8,11 +8,29 @@ import {Button} from '@jahia/moonstone';
 import {useTranslation} from 'react-i18next';
 import {adaptSectionToDisplayableRows} from './AutomaticOrdering.utils';
 import {getDisplayedRows} from './AutomaticOrdering.utils';
+import {withStyles} from '@material-ui/core';
+import {Clear} from '@material-ui/icons';
+import {IconButton} from '@jahia/design-system-kit';
 
-export const AutomaticOrderingCmp = ({formik: {values}}) => {
+const styles = theme => ({
```

Comment:
it's great if you can move the style part to scss file, it's more easy to maintain

Reply:
will do in next PR